openapi/openapi.yaml

```diff
@@ -896,6 +885,13 @@ components:
           description: Namespace brief description
           type: string
           maxLength: 256
+        resource_page_src:
+          description: Namespace resource page in Markdown format.
+          type: string
+        resource_page_html:
```

Comment:
Since we're rendering the mardown in the UI, we can get rid of this field.

Reply:
Are we. Then yes, sure.